jaeger-core/src/test/java/io/jaegertracing/ConfigurationTest.java

```diff
@@ -184,11 +203,11 @@ public void testPropagationB3Only() {
     System.setProperty(Configuration.JAEGER_PROPAGATION, "b3");
     System.setProperty(Configuration.JAEGER_SERVICE_NAME, "Test");
 
-    long traceId = 1234;
-    long spanId = 5678;
+    TraceId traceId = new TraceId(1234);
```

Comment:
Please try a bigger TraceId value, preferably > 64 bits. I'd like to see propagation works for large numbers too.

Reply:
I just noticed that we propagate 128 bits ids in B3 formats. Not sure how the receiver of the trace context (ie the brave library) will react to this.